### PR TITLE
[MM-32799] Add About Section

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Add About SubSections

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-32794

#### Release Note

```release-note
None
```

### CODE REVIEW

File: model/role.go [hunk 1]
```diff
@@ -152,7 +159,7 @@ func init() {
 	}
 
 	SystemManagerDefaultPermissions = []string{
-		PERMISSION_SYSCONSOLE_READ_ABOUT.Id,
+		PERMISSION_SYSCONSOLE_READ_ABOUT_EDITION_AND_LICENSE.Id,
```
Comment (maxerenberg) [inline model/role.go:162]: Is the system manager not supposed to have PERMISSION_SYSCONSOLE_WRITE_ABOUT_EDITION_AND_LICENSE?

File: model/role.go [hunk 2]
```diff
@@ -507,6 +514,7 @@ func (r *Role) IsValidWithoutId() bool {
 	for _, permission := range r.Permissions {
 		permissionValidated := check(AllPermissions, permission) || check(DeprecatedPermissions, permission)
 		if !permissionValidated {
+			fmt.Printf("\n\n\n\n\n\n\n\n %+v \n\n\n\n\n\n\n\n\n", permission)
```
Comment (maxerenberg) [inline model/role.go:517]: What's this?

File: model/role.go (hunk 1 above)
Comment (sbishel) [reply model/role.go:162]: According to this doc, https://mattermost.atlassian.net/wiki/spaces/EN/pages/463798295/New+Admin+Roles+Special+Admins
System Manager is RO on `Edition and License`

**Review by sbishel — APPROVED** [review]
Just remove the `fmt.Print` statement that Max pointed out.

File: model/role.go (hunk 2 above)
Comment (hahmadia) [reply model/role.go]: @maxerenberg A mistake 😆

**Review by maxerenberg — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]

**Pull request merged**